README.md

```diff
@@ -195,14 +195,21 @@ Update browser settings:
     - In the mobile app Settings page, tap "Search" within the "General" section. There should be an option titled "Add Search Engine" to select. It should prompt you to enter a title and search query url - use the following elements to fill out the form:
       - Title: "Whoogle"
       - URL: "https://\<your whoogle url\>/search?q=%s"
-  - Firefox (Android)
+  - Firefox <79.0.0 (Android)
```

Comment:
I think I'd prefer to combine this and the one below into one `Firefox (Android)` bullet:

```suggestion
  - Firefox (Android)
    - Version <79.0.0
      - ...
    - Version >=79.0.0
      - ...
```